Bukkit/src/main/java/me/fixeddev/commandflow/bukkit/part/GameModePart.java

```diff
@@ -69,11 +66,16 @@ public GameModePart(String name) {
         }
 
         String possibleGameModeName = next.toUpperCase();
+        List<String> suggestions = new ArrayList<>();
+
+        for (GameMode gameMode : GameMode.values()) {
+            String name = gameMode.name().toLowerCase();
+            if (possibleGameModeName.length() == 0 || name.startsWith(possibleGameModeName)) {
```

Comment:
isEmpty()

Reply:
ready

Reply:
`GameMode.values()` could be replaced with already cached values of Map GAMEMODE_ALIASES

Reply:
Or directly with the aliases

Reply:
I forgot to say, but I've already changed it